src/storage/table.cc

```diff
@@ -152,14 +163,6 @@ void Table::UpdateTTL() {
             }
         }
     }
-    auto table_meta = std::atomic_load_explicit(&table_meta_, std::memory_order_acquire);
-    auto new_table_meta = std::make_shared<::openmldb::api::TableMeta>(*table_meta);
-    new_table_meta->clear_column_key();
-    for (auto& index : indexs) {
-        auto column_key = new_table_meta->add_column_key();
-        column_key->CopyFrom(index->GenColumnKey());
-    }
-    std::atomic_store_explicit(&table_meta_, new_table_meta, std::memory_order_release);
```

Comment:
no need to update the `table_meta_` in `UpdateTTL()`?

Reply:
have moved to `SetTTL` function